kai_mcp_solution_server/tests/mcp_client.py

```diff
@@ -454,8 +461,8 @@ def patched_async_client_init(self, *args, **kwargs):
             # Setup STDIO transport
             server_params = StdioServerParameters(
                 command="python",
-                args=[str(args.server_path), "--transport", "stdio"],
-                cwd=args.server_path.parent,
+                args=["-m", "kai_mcp_solution_server", "--transport", "stdio"],
+                cwd=str(args.server_path),
```

Comment:
I modified `args.server_path` to be the `__main__.py` file of the server itself, not the directory. This should fix the failing test

```suggestion
                cwd=str(args.server_path.parent),
```